diff --git a/base/metrics/histogram_functions.h b/base/metrics/histogram_functions.h
index 52ea28ce01ba8d59ec13974f83796ac2d9c8902b..8ff0f4d4e84fd4e1f187cb096c43d7b999fee429 100644
--- a/base/metrics/histogram_functions.h
+++ b/base/metrics/histogram_functions.h
@@ -86,6 +86,7 @@ void UmaHistogramEnumeration(const StringType& name, T sample) {
                 "Enumeration's kMaxValue is out of range of "
                 "LinearHistogram::kBucketCount_MAX. Use a sparse histogram "
                 "instead.");
+  BRAVE_HISTOGRAM_FUNCTIONS_UMA_HISTOGRAM_ENUMERATION
   DCHECK_LE(static_cast<uintmax_t>(sample),
             static_cast<uintmax_t>(T::kMaxValue));
   return UmaHistogramExactLinear(name, static_cast<int>(sample), kBucketCount);
