diff --git a/chrome/browser/resources/settings/site_settings/site_settings_browser_proxy.ts b/chrome/browser/resources/settings/site_settings/site_settings_browser_proxy.ts
index 2ce1adeaa6da8eb7c6da5c9d337f1ef323cfdd4b..58c84413f183ef315e3bf81a946572bc0d36e92c 100644
--- a/chrome/browser/resources/settings/site_settings/site_settings_browser_proxy.ts
+++ b/chrome/browser/resources/settings/site_settings/site_settings_browser_proxy.ts
@@ -83,6 +83,7 @@ export interface RawSiteException {
   description?: string;
   setting: ContentSetting;
   source: SiteSettingSource;
+  braveCookieType?: undefined | "shields down" | "shields settings" | "google sign-in";  // See BraveGetExceptionForPage (site_settings_helper.cc)
 }
 
 /**
@@ -100,6 +101,7 @@ export interface SiteException {
   description?: string;
   enforcement: chrome.settingsPrivate.Enforcement|null;
   controlledBy: chrome.settingsPrivate.ControlledBy;
+  braveCookieType?: RawSiteException['braveCookieType'];
 }
 
 /**
