diff --git a/chrome/browser/ui/android/toolbar/BUILD.gn b/chrome/browser/ui/android/toolbar/BUILD.gn
index 6f6fab210a6a9c72e57014c64ed12229e48f0da4..c1cb94d1828c69c298b0824d379e7664f7a3b988 100644
--- a/chrome/browser/ui/android/toolbar/BUILD.gn
+++ b/chrome/browser/ui/android/toolbar/BUILD.gn
@@ -138,6 +138,7 @@ android_library("core_java") {
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/HeightTransitionHandler.java",
     "java/src/org/chromium/chrome/browser/toolbar/top/tab_strip/TabStripTransitionCoordinator.java",
   ]
+  sources += brave_browser_ui_android_toolbar_sources
 
   deps = [
     ":java_resources",
@@ -351,6 +352,7 @@ android_resources("java_resources") {
     "//components/browser_ui/styles/android:java_resources",
     "//ui/android:ui_java_resources",
   ]
+  deps += [ "//brave/browser/ui/android/toolbar:java_resources" ]
 }
 
 robolectric_library("junit") {
