diff --git a/chrome/browser/ui/webui/settings/people_handler.cc b/chrome/browser/ui/webui/settings/people_handler.cc
index f4f0a0153148c8f5beb7d2215b4758093d49a644..c245920fef1ea0ce6edc6837e1a0ba9f6c38a1c0 100644
--- a/chrome/browser/ui/webui/settings/people_handler.cc
+++ b/chrome/browser/ui/webui/settings/people_handler.cc
@@ -1150,6 +1150,7 @@ void PeopleHandler::OnSyncShutdown(syncer::SyncService* sync_service) {
 
 void PeopleHandler::BeforeUnloadDialogCancelled() {
   // The before unload dialog is only shown during the first sync setup.
+  if (false)
   DCHECK(IdentityManagerFactory::GetForProfile(profile_)->HasPrimaryAccount(
       signin::ConsentLevel::kSync));
   syncer::SyncService* service = GetSyncService();
@@ -1199,6 +1200,7 @@ base::DictValue PeopleHandler::GetSyncStatusDictionary() const {
       service && !disallowed_by_policy && service->IsSetupInProgress() &&
           !service->GetUserSettings()->IsInitialSyncFeatureSetupComplete() &&
           identity_manager->HasPrimaryAccount(signin::ConsentLevel::kSync));
+  BRAVE_GET_SYNC_STATUS_DICTIONARY
 
   SyncStatusLabels status_labels = GetSyncStatusLabelsForSettings(
       SyncServiceFactory::GetForProfile(profile_));
@@ -1409,6 +1411,7 @@ void PeopleHandler::MaybeMarkSyncConfiguring() {
 }
 
 bool PeopleHandler::IsProfileAuthNeededOrHasErrors() {
+  return false;
   return !IdentityManagerFactory::GetForProfile(profile_)->HasPrimaryAccount(
              signin::ConsentLevel::kSync) ||
          SigninErrorControllerFactory::GetForProfile(profile_)->HasError();
