diff --git a/chrome/browser/resources/settings/route.ts b/chrome/browser/resources/settings/route.ts
index 7efda4aabfe462bf5ccad37de5029357bf4fe350..2d094a72cb378f92d9a73e011ba881f54479fcfd 100644
--- a/chrome/browser/resources/settings/route.ts
+++ b/chrome/browser/resources/settings/route.ts
@@ -9,6 +9,7 @@ import {pageVisibility} from './page_visibility.js';
 import type {SettingsRoutes} from './router.js';
 import {Route, Router} from './router.js';
 
+import addBraveRoutes from './brave_routes.js'
 /**
  * Add all of the child routes that originate from the privacy route,
  * regardless of whether the privacy section under basic or advanced.
@@ -324,6 +325,7 @@ function createRoutes(): SettingsRoutes {
         loadTimeData.getString('performancePageTitle'));
   }
 
+  addBraveRoutes(r);
   return r as unknown as SettingsRoutes;
 }
 
@@ -357,7 +359,7 @@ export let routes: SettingsRoutes = Router.getInstance().getRoutes();
 // Returns the "effective" route when at chrome://settings/.
 export function getTopLevelRoute(): Route {
   if (!loadTimeData.getBoolean('isGuest')) {
-    return routes.PEOPLE;
+    return routes.GET_STARTED;
   }
 
   let guestTopLevelRoute = routes.SEARCH;
