diff --git a/chrome/browser/ui/browser_window/public/browser_window_features.h b/chrome/browser/ui/browser_window/public/browser_window_features.h
index 917cb72be85a0528a5be2b594078177a093dc4ae..7f3e7eb5c31fa899c7cfb3471ca9947654e18f61 100644
--- a/chrome/browser/ui/browser_window/public/browser_window_features.h
+++ b/chrome/browser/ui/browser_window/public/browser_window_features.h
@@ -229,7 +229,7 @@ class SkillsUiWindowController;
 class BrowserWindowFeatures {
  public:
   BrowserWindowFeatures();
-  ~BrowserWindowFeatures();
+  virtual ~BrowserWindowFeatures();
 
   BrowserWindowFeatures(const BrowserWindowFeatures&) = delete;
   BrowserWindowFeatures& operator=(const BrowserWindowFeatures&) = delete;
