diff --git a/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java b/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java
index 847921fb1a41e4907c10d654651ec0db6bf610ae..ada2bcbec7dc9b08976bb79a3bfb98b08073cab2 100644
--- a/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java
+++ b/chrome/browser/ui/messages/android/java/src/org/chromium/chrome/browser/ui/messages/snackbar/SnackbarManager.java
@@ -405,7 +405,7 @@ public class SnackbarManager
             boolean viewChanged = true;
             if (mView == null) {
                 mView =
-                        new SnackbarView(
+                        new BraveSnackbarView(
                                 mActivity,
                                 this,
                                 currentSnackbar,
