diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index ad16de65f19c1e8cb1a5354644a14ff412d65433..dac9d46fa5d2e1bcc64e3aabc3752dc566ea4162 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4800,6 +4800,7 @@ const FeatureEntry::Choice kConnectionAllowlistsChoices[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
