diff --git a/chrome/browser/profiles/profile_attributes_entry.cc b/chrome/browser/profiles/profile_attributes_entry.cc
index 1a537b5ca754d2c711a16bd5ca8d6a1bfa2fe424..ed9c2c5349ea3fbf02958cfc857ebffc68880b1a 100644
--- a/chrome/browser/profiles/profile_attributes_entry.cc
+++ b/chrome/browser/profiles/profile_attributes_entry.cc
@@ -1057,6 +1057,7 @@ bool ProfileAttributesEntry::ClearValue(const char* key) {
 
 // This method should be periodically pruned of year+ old migrations.
 void ProfileAttributesEntry::MigrateObsoleteProfileAttributes() {
+  BRAVE_PROFILE_ATTRIBUTES_ENTRY_MIGRATE_OBSOLETE_PROFILE_ATTRIBUTES
   // Added 3/2023.
   ClearValue(kAccountCategories);
   // Added 2/2026.
