diff --git a/chrome/browser/content_settings/BUILD.gn b/chrome/browser/content_settings/BUILD.gn
index 9c461e5ceac0c7a47bc83bef152d2b8e75ab74c6..0f58f8fa8b27edc95933d0adb920b6d0ca195c15 100644
--- a/chrome/browser/content_settings/BUILD.gn
+++ b/chrome/browser/content_settings/BUILD.gn
@@ -131,6 +131,7 @@ source_set("impl") {
   if (enable_pdf) {
     deps += [ "//chrome/browser/pdf" ]
   }
+  import("//brave/browser/content_settings/sources.gni") deps += brave_browser_content_settings_deps
 }
 
 # Fixes circular deps with different components. For example -
@@ -338,6 +339,7 @@ source_set("unit_tests") {
   if (is_android) {
     sources += [ "javascript_optimizer_provider_android_unittest.cc" ]
     deps += [ "//chrome/browser/android:tabs_public" ]
+    public_deps += [ "//brave/extensions:common" ]
   } else {
     sources += [
       "generated_cookie_prefs_unittest.cc",
