diff --git a/chrome/browser/hub/internal/BUILD.gn b/chrome/browser/hub/internal/BUILD.gn
index e129ca79741897e8b2aaec2d7ce40157a56738dc..e1996b3953953efd20394b9593ac7f25dee4575c 100644
--- a/chrome/browser/hub/internal/BUILD.gn
+++ b/chrome/browser/hub/internal/BUILD.gn
@@ -106,6 +106,7 @@ android_library("java") {
     "//ui/edge_to_edge:java",
     "//url:url_java",
   ]
+  sources += brave_browser_hub_internal_java_sources deps += brave_browser_hub_internal_java_deps
 }
 
 android_resources("java_resources") {
