diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index 627770ec433714b785645ea380f73f10e66b0720..aa12e98b6935d60d953fd2f164fd56cc006f1d52 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1321,3 +1321,4 @@ if (enable_screen_capture) {
     "java/src/org/chromium/chrome/browser/media/MediaCapturePickerTabObserver.java",
   ]
 }
+chrome_java_sources -= ["java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java",]
