diff --git a/chrome/browser/ui/tabs/tab_strip_model.h b/chrome/browser/ui/tabs/tab_strip_model.h
index 6f36d788e89320d26fe123684a0c56396733694d..ae30d8e0cbe1832b24c41e810530b96d97dc54af 100644
--- a/chrome/browser/ui/tabs/tab_strip_model.h
+++ b/chrome/browser/ui/tabs/tab_strip_model.h
@@ -211,6 +211,7 @@ class TabStripModel {
   TabStripModel(const TabStripModel&) = delete;
   TabStripModel& operator=(const TabStripModel&) = delete;
 
+  virtual
   ~TabStripModel();
 
   // Retrieves the TabStripModelDelegate associated with this TabStripModel.
