diff --git a/components/search_engines/default_search_manager.cc b/components/search_engines/default_search_manager.cc
index 7ad870db99f59e7b7246549ec8ff6e6e55217364..03125bbe725d030ffbcb6c0b4221248b06b67426 100644
--- a/components/search_engines/default_search_manager.cc
+++ b/components/search_engines/default_search_manager.cc
@@ -360,6 +360,7 @@ void DefaultSearchManager::LoadDefaultSearchEngineFromPrefs() {
   // Check if default search preference is overridden by extension.
   if (pref->IsExtensionControlled()) {
     extension_default_search_ = std::move(turl_data);
+  LOADDEFAULTSEARCHENGINEFROMPREFS_BRAVE
   } else {
     prefs_default_search_.SetAndReconcile(std::move(turl_data));
   }
