diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 4f9e04496cf9a179702cef7b85f5bdc7483e7aea..5b722a963ff7d4f8bc910ce974f9fa91dfc3a818 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -352,6 +352,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
