diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 6fb162e7f477fabde5a83de7bf68b87f1708f274..8c98fdffba0159a7b3a3c37d27fbdbb80153fab8 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3760,6 +3760,7 @@ source_set("browser") {
       "surface_embed/surface_embed_connector_impl.h",
     ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
