diff --git a/content/browser/renderer_host/navigation_controller_impl.cc b/content/browser/renderer_host/navigation_controller_impl.cc
index 16248f8c246e209fcf0bc96d8829fe201f6eb049..464f44497da4b8ff23bc7f3c83ae0465751130b0 100644
--- a/content/browser/renderer_host/navigation_controller_impl.cc
+++ b/content/browser/renderer_host/navigation_controller_impl.cc
@@ -455,6 +455,7 @@ void RewriteUrlForNavigation(const GURL& original_url,
   *url_to_load = *virtual_url = original_url;
   BrowserURLHandlerImpl::GetInstance()->RewriteURLIfNecessary(
       url_to_load, browser_context, reverse_on_redirect);
+  MaybeRewriteVirtualURL(virtual_url);
 }
 
 #if DCHECK_IS_ON()
