diff --git a/net/http/transport_security_state.cc b/net/http/transport_security_state.cc
index 9dc4e462def223ac6e2d8c2e1e19efb637d614b9..1629bf6c43b30bc8dd07aa5a27972b1c83b7af92 100644
--- a/net/http/transport_security_state.cc
+++ b/net/http/transport_security_state.cc
@@ -237,6 +237,7 @@ TransportSecurityState::TransportSecurityState(
 // others don't end up with pins that cannot be easily updated.
 #if !BUILDFLAG(GOOGLE_CHROME_BRANDING) || BUILDFLAG(IS_IOS)
   enable_static_pins_ = false;
+  BRAVE_ENABLE_STATIC_PINS
 #endif
   // Check that there no invalid entries in the static HSTS bypass list.
   for (auto& host : hsts_host_bypass_list) {
