diff --git a/third_party/blink/public/common/BUILD.gn b/third_party/blink/public/common/BUILD.gn
index 6936a114dcf4037b97aec2628b5209f99fdbf6c8..37b4629ae58495ff6cc299e2bfa1d87b1b7b331f 100644
--- a/third_party/blink/public/common/BUILD.gn
+++ b/third_party/blink/public/common/BUILD.gn
@@ -387,6 +387,7 @@ source_set("headers") {
   if (is_mac) {
     sources += [ "sandbox_support/sandbox_support_mac.h" ]
   }
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_public_common_deps
 }
 
 if (is_android || is_win) {
