diff --git a/third_party/blink/common/BUILD.gn b/third_party/blink/common/BUILD.gn
index c8e90d61bc16fabb58ebb1313ed7bf03173ad15b..5cabd19891f1dc578fdbaa6f8cfa2d0b4404345d 100644
--- a/third_party/blink/common/BUILD.gn
+++ b/third_party/blink/common/BUILD.gn
@@ -351,6 +351,7 @@ source_set("common") {
   if (is_chromeos || is_android || is_win || is_castos || is_fuchsia) {
     defines = [ "ENABLE_PROTECTED_MEDIA_IDENTIFIER_PERMISSION" ]
   }
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 test("blink_common_unittests") {
@@ -497,6 +498,7 @@ source_set("common_unittests_sources") {
   # TODO(crbug.com/40031409): Fix code that adds exit-time destructors and
   # enable the diagnostic by removing this line.
   configs += [ "//build/config/compiler:no_exit_time_destructors" ]
+  import("//brave/components/brave_wallet/sources.gni") deps += brave_blink_common_deps
 }
 
 group("common_unittests_data") {
