diff --git a/ui/base/BUILD.gn b/ui/base/BUILD.gn
index 7cdab4235dca2ebdbdf77c6324df14f3094750d4..d81cd448f05185e69ba6b78a40210e44c3436fef 100644
--- a/ui/base/BUILD.gn
+++ b/ui/base/BUILD.gn
@@ -502,6 +502,7 @@ component("base") {
     "//ui/webui/resources:resources_grit",
     "//url",
   ]
+  import("//brave/chromium_src/ui/base/sources.gni") deps += brave_chromium_src_ui_base_deps
 
   if (is_debug || dcheck_always_on) {
     deps += [ "//third_party/re2" ]
