diff --git a/src/codegen/compiler.cc b/src/codegen/compiler.cc
index 848ee19058d2a5f618d0f46dd9b7c79efdddb22b..a34ca6fb9ac891c13a965806db0411eadd405f2b 100644
--- a/src/codegen/compiler.cc
+++ b/src/codegen/compiler.cc
@@ -3451,6 +3451,7 @@ MaybeDirectHandle<JSFunction> Compiler::GetFunctionFromEval(
     }
   }
   CHECK(is_compiled_scope.is_compiled());
+  BRAVE_COMPILER_GET_FUNCTION_FROM_EVAL
 
   return result;
 }
